packages/core/admin/server/src/bootstrap.ts

```diff
@@ -145,6 +145,13 @@ export default async ({ strapi }: { strapi: Core.Strapi }) => {
     jwtOptions: options,
   });
 
+  const isProduction = process.env.NODE_ENV === 'production';
+  const adminCookieSecure = strapi.config.get('admin.auth.cookie.secure');
+  if (isProduction && adminCookieSecure === false) {
+    // TODO
+    strapi.log.warn('PLACEHOLDER WARNING: Session cookie secure');
```

Comment:
```suggestion
    strapi.log.warn('Server is in production mode, but admin.auth.cookie.secure has been set to false');
```
Maybe something like that?